inventory/byo/hosts.origin.example

```diff
@@ -128,6 +128,9 @@ openshift_release=v3.6
 
 # Alternate image format string, useful if you've got your own registry mirror
 #oreg_url=example.com/openshift3/ose-${component}:${version}
+# Specifically set oreg_url on the master or on nodes
+#oreg_url_master=example.com/openshift3/ose-${component}:${version}
```

Comment:
Based on the decision we make in my other comment lets make sure to mention the variable precedence here so it's clear.